catalog/transformer/catalog-transformer-resource/src/main/java/ddf/catalog/transformer/resource/ResourceMetacardTransformer.java

```diff
@@ -73,6 +73,9 @@ public BinaryContent transform(Metacard metacard, Map<String, Serializable> argu
         }
 
         String id = metacard.getId();
+        if(StringUtils.isNotEmpty(metacard.getResourceSize())) {
```

Comment:
This might be a bit pedantic, but it seems wrong from a scoping perspective to put this immediately after the declaration of the variable `id` when it doesn't use that variable.